pkg/controller/windowsmachine/windowsmachine_controller.go

```diff
@@ -313,3 +363,45 @@ func (r *ReconcileWindowsMachine) validateUserData() error {
 	}
 	return nil
 }
+
+// isAllowedDeletion determines if the number of machines after deletion of the given machine doesn`t fall below the
+// minHealthyCount
+func (r *ReconcileWindowsMachine) isAllowedDeletion(machine *mapi.Machine) bool {
+	machinesetName := machine.OwnerReferences[0].Name
+
+	machines := &mapi.MachineList{}
+	err := r.client.List(context.TODO(), machines,
+		client.MatchingLabels(map[string]string{"machine.openshift.io/os-id": "Windows"}))
```

Comment:
can we make the label as a constant?

Reply:
Same as [here](https://github.com/openshift/windows-machine-config-operator/pull/101#discussion_r486633614)